www/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "src",
+    "rootDir": "../",
```

Comment:
I'm curious as to what happened before this was changed.

Reply:
Without this change, I was getting errors like this when running `./node_modules/.bin/tsc --project www --noEmit` from the main `paragon` root:

```
www/src/components/AutoToc.tsx:5:24 - error TS6059: File '/Users/opencraft/Documents/tutor-devstack/paragon/src/index.ts' is not under 'rootDir' '/Users/opencraft/Documents/tutor-devstack/paragon/www/src'. 'rootDir' is expected to contain all source files.
  The file is in the program because:
    Imported via '~paragon-react' from file '/Users/opencraft/Documents/tutor-devstack/paragon/www/src/components/AutoToc.tsx'
```

However, it seems like the commit 688d212f49c3c2950b54c10e57e86ac0804eb2c6 resolved this in another way, by changing from `index.ts` to `index.d.ts` so this change to `www/tsconfig.json`'s `rootDir` is no longer needed. I'll just leave it as it was.